vbjax/neural_mass.py

```diff
@@ -1,33 +1,38 @@
 import collections
 import jax.numpy as np
 
+# Jansen-Rit
 
 JRTheta = collections.namedtuple(
     typename='JRTheta',
     field_names='A B a b v0 nu_max r J a_1 a_2 a_3 a_4 mu I'.split(' '))
 
 jr_default_theta = JRTheta(
-    A=3.25, B=22.0, a=0.1, b=0.05, v0=5.52, nu_max=0.0025, 
```

Comment:
please turn off your autoformatter.  don't worry about this commit but in the future, try to avoid mixing whitespace changes with content.